core/src/test/java/org/apache/accumulo/core/metadata/schema/SortSkewTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.core.metadata.schema;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class SortSkewTest {
```

Comment:
Since this class is hashing data for persistence, I think it wold be nice if it ensured the generated hash codes were the same across time.   This could be done by hashing a few random data string to see what they currently hash to and then using them as literals in the test.  Something like the following.

```java
public void verifyStable() {
  assertEquals("abcdef01", SortSkew.getCode("/dir/path1"));
  assertEquals("12345678", SortSkew.getCode("/dir/path2"));
}
```